### ISSUE TITLE
any updates?

### ISSUE THREAD

**Issue by vikram06** [muesli/go-crashcourse#1]
Any updates ? we are waiting :)

**Comment by muesli** [comment]
Yes, I have more work in the pipeline! However I realized it makes little sense to keep posting them every other week, so instead I'll do a full first course with a couple of videos and examples and post them once I'm finished.
